src/app/main-menu/main-menu.component.html

```diff
@@ -14,14 +14,12 @@ <h1>Capture network</h1>
     <mat-nav-list>
       <a mat-list-item [routerLink]="'/'"> Home </a>
       <app-main-menu-item mat-list-item *ngFor="let game of gameList" [gameDescription]="game"
-        [routerLink]="['/game', game.GAME_ID]">
+        [routerLink]="['/game', game.game_id]" [queryParams]="{bot1: game.bot_1.name, bot2: game.bot_2.name}">
```

Comment:
This line is little too long in my opinion.

Reply:
Thats how default formatter format html.
I changed line size so it will be fixed when I polish main menu.